grpc-spring-boot-starter/src/main/java/org/lognet/springboot/grpc/GRpcServerRunner.java

```diff
@@ -53,6 +60,31 @@ public void run(String... args) throws Exception {
 
     }
 
+    private ServerServiceDefinition bindInterceptors(BindableService bindableService) {
+        Class<? extends ServerInterceptor>[] interceptorClasses = bindableService.getClass().getAnnotation(GRpcService.class).interceptors();
+
+        if(interceptorClasses.length == 0) {
+            bindableService.bindService();
+        }
+
+        List<ServerInterceptor> interceptors = buildInterceptors(interceptorClasses);
+        return ServerInterceptors.intercept(bindableService, interceptors);
+    }
+
+    private List<ServerInterceptor> buildInterceptors(Class<? extends ServerInterceptor>[] classes) {
+        List<ServerInterceptor> interceptors = new ArrayList<>();
+
+        for(Class<? extends ServerInterceptor> clazz : classes) {
+            try {
+                interceptors.add(clazz.newInstance());
```

Comment:
Don't you think it might be useful to instantiate Interceptor  via Spring as Spring Bean  and leverage from all the  goodies (DI, life-cycle management, configuration support, etc ... )?
```
interceptors.add(applicationContext.getBean(clazz))
```